pkg/reconciler/v1alpha1/taskrun/entrypoint/entrypoint_test.go

```diff
@@ -79,6 +74,62 @@ func TestAddEntrypoint(t *testing.T) {
 	}
 }
 
+func TestGetArgs(t *testing.T) {
+	// first step
+	// multiple commands
+	// no args
+	for _, c := range []struct {
+		desc         string
+		stepNum      int
+		commands     []string
+		args         []string
+		expectedArgs []string
+	}{{
+		desc:     "Args for first step",
+		stepNum:  0,
+		commands: []string{"echo"},
+		args:     []string{"hello", "world"},
+		expectedArgs: []string{
+			"-wait_file", "",
+			"-post_file", "/tools/0",
```

Comment:
used `/tools` because it was already there from the log writing entrypoint that got removed. Is `builder` created anyway, or just as part of a git resource?

Reply:
moved `tools` to `/builder/tools`